enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -29,6 +29,7 @@ const (
 	MetricEligibleGauge             = namespace + "eligible"
 	MetricPresetHardLimitedGauge    = namespace + "preset_hard_limited"
 	MetricLastUpdatedGauge          = namespace + "metrics_last_updated"
+	MetricReconciliationPausedGauge = namespace + "reconciliation_paused"
```

Comment:
Paused is difficult to reason about; I'd suggest inverting and make it `_running` so that a good value is 1 and a bad value is 0.